addons/account_check_printing/models/account_journal.py

```diff
@@ -39,7 +39,7 @@ def _compute_check_next_number(self):
         for journal in self:
             sequence = journal.check_sequence_id
             if sequence:
-                journal.check_next_number = sequence.get_next_char(sequence.number_next_actual)
+                journal.check_next_number = sequence.next_by_id()
```

Comment:
@william-andre @guva-odoo Is not the next_by_id consuming and the get_next_char not?  I find it very weird to consume a sequence number in a compute.